source/_integrations/zha.markdown

```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
 
 There is also support for grouping of lights, switches, and fans (i.e. support for commanding device groups as entities). At least two entities must be added to a group before the group entity is created. As well as support for binding/unbinding (i.e. bind a remote to a lightbulb or group).
 
+## Introduction and dictionary (terminology vocabulary)
+
+ZHA is a native “Zigbee host application” for Home Assistant that via a Zigbee Coordinator adapter will setup and allow control of a single “Zigbee network” (using mesh topology). Once installed ZHA you will be able to directly join/pair most types of “Zigbee Router Device” and “Zigbee End Device” products to that Zigbee network, regardless of product manufacturer and brand.
```

Comment:
A user coming and reading this for the first time, will have no clue what all these things even mean.